packages/sitecore-jss/src/native-fetcher.ts

```diff
@@ -102,4 +105,16 @@ export class NativeDataFetcher {
 
     return init;
   }
+
+  private extractHeaders(incomingHeaders: HeadersInit = {}) {
```

Comment:
Make this protected (in case anyone needs to access/override in derived class) and rename so it's clear that only for debug purposes?
```suggestion
  protected extractDebugHeaders(incomingHeaders: HeadersInit = {}) {
```